f2/CHANGELOG.md

```diff
@@ -1,24 +1,52 @@
 # Changelog for report a cybercrime product
 
-## V1.1.0 (staging: 2020-03-16)
+## V1.1.0 (2020-04-14)
```

Comment:
V1.2.0 ?

Reply:
We never actually got the previous 1.1.0 staging out the door (ie didn't put it to prod) so I figured our current release was 1.1.0 instead.

Reply:
So should we stick to 1.1.0 in initialState ?

Reply:
ah, I see your point! sorry, slow this morning. So the 1.2.0 in initialState is what the code is going forward. ie master from now on will be 1.2.0 rather than 1.1.0. I though doing it this way would make it easier to do releases, ie wouldn't have to change the counter. but... it's confusing. And what if the next release was "1.1.1" or "2.0.0". Alright, I'm convinced, let me change the initialState version to something else. Possibly something like "1.1.0-dev" or something. Then we change it to 1.2.0 or whatever when we actually cut the next release.

Reply:
I see, makes sense. W'ere working on 1.2.0, but we are releasing the changelog from 1.1.0. Makes sense to me either way